api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -191,6 +191,9 @@ message ResponseFlags {
       REASON_UNSPECIFIED = 0;
       // The request was denied by the external authorization service.
       EXTERNAL_SERVICE = 1;
+      // The request was denied because a header value failed a :ref:`strict header check
+      // <envoy_api_field_config.filter.http.router.v2.Router.strict_check_headers>` failed.
+      STRICT_HEADER_CHECK_FAILED = 2;
```

Comment:
Not sure if this belongs here, this seems to be dealing with authorization of an endpoint, not about whether the request is valid. If there isn't a similar flag for "invalid request reason" we can probably just drop this

Reply:
I think you're right. It looks like I should instead move this to be a `bool` field to match treatment of [the other response flags in accesslog.proto](https://github.com/envoyproxy/envoy/blob/6eec18ce14861c7ba4b92e6faea162cf41cb268a/api/envoy/data/accesslog/v2/accesslog.proto#L152-L186)